src/online-members/script.js

```diff
@@ -136,6 +136,12 @@ async function generateMemberTaskData(username) {
     return;
   }
 
+  if (!username) {
+    throw new Error(
+      'Some error occurred while getting user data, please try again or contact admin',
+    );
+  }
+
```

Comment:
You have handled that if the username is the null in the "addEventListenerToMembersList" function already before calling the "getTaskDataContent" function there, then why are you checking this again here?

Reply:
Yes, right this has been already handled, will remove it.
Thanks for pointing it out :smiley: